### ISSUE TITLE
Blocks: cannot drag and drop fullwidth blocks

### ISSUE THREAD

**Issue by phpbits** [WordPress/gutenberg#7914]
Hi,

I'm testing blocks and it seems that when cover image or any other blocks with full width doesn't have option to be dragged or reposition when on full width mode.

**Comment by designsimply** [comment]
Hi! Tested and confirmed using the following testing steps:

1. Install a theme with support for wide blocks (e.g. [Gutenberg Starter Theme](https://github.com/WordPress/gutenberg-starter-theme), [Navi](https://github.com/jasmussen/navi), [Atomic Blocks](https://wordpress.org/themes/atomic-blocks/)).
1. Create a post.
1. Add a cover block image and set it to full width.
1. Try to drag the full-width cover image block to a new location inside the editor.

<strong>Result:</strong> cannot drag and drop wide blocks using full width because the draggable handles are hidden left or right.

<strong>Video:</strong> [28s](https://cloudup.com/czKbQeY3Q8l)
<sup>Tested with WordPress 4.9.7 and Gutenberg 3.2.0 using Firefox 61.0.1 on macOS 10.13.5.</sup>

![draggable-handles-hidden-for-fullwidth-wide-blocks](https://user-images.githubusercontent.com/1119271/42649922-3e17becc-85c8-11e8-9650-6a4565d42d90.png)

**Comment by mtias** [comment]
Closing in favor of https://github.com/WordPress/gutenberg/issues/7114

**Comment by couponrim** [comment]
![Uploading Annotation 2019-01-28 031045.jpg…]()
I have disabled all of the plugins but still the same.

**Comment by designsimply** [comment]
@couponrim you have posted in a closed GitHub issue that is about not being able to use the drag handle to move a full-width block, however, it is not clear from your comment you are having the same problem. 

I've created a video to show what issue 7914 is about to illustrate: [1m41s](https://cloudup.com/cFtaBsUgjOj)
<sup>Tested with Firefox 64.0.2, Chrome 71.0.3578.98, and Safari 12.0.2 on macOS 10.13.6.</sup>

It appears you may need some help with troubleshooting before posting to GitHub. If you are still having trouble with this issue or with another similar issue, would you please post a short explanation and a numbered list of your testing steps at https://wordpress.org/support/forum/how-to-and-troubleshooting/ and ask for help with troubleshooting there first? They should be able to help guide you there and if you work in that forum to identify whether or not something is a bug and if you determine there is a bug then that would be the time to come back to GitHub and create a new issue with the findings. Please include details such as your WordPress version and your OS and browser version as those things may be relevant for your case.

**Comment by couponrim** [comment]
> @couponrim you have posted in a closed GitHub issue that is about not being able to use the drag handle to move a full-width block, however, it is not clear from your comment you are having the same problem.
> 
> I've created a video to show what issue 7914 is about to illustrate: [1m41s](https://cloudup.com/cFtaBsUgjOj)
> Tested with Firefox 64.0.2, Chrome 71.0.3578.98, and Safari 12.0.2 on macOS 10.13.6.
> 
> It appears you may need some help with troubleshooting before posting to GitHub. If you are still having trouble with this issue or with another similar issue, would you please post a short explanation and a numbered list of your testing steps at https://wordpress.org/support/forum/how-to-and-troubleshooting/ and ask for help with troubleshooting there first? They should be able to help guide you there and if you work in that forum to identify whether or not something is a bug and if you determine there is a bug then that would be the time to come back to GitHub and create a new issue with the findings. Please include details such as your WordPress version and your OS and browser version as those things may be relevant for your case.

Thank you so much for your helps.
